test/Bridges/Constraint/ltgt_to_interval.jl

```diff
@@ -0,0 +1,130 @@
+        (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock, [100.0, 0.0]),
+        (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock, [100.0, -100.0]))
+    MOIT.linear6test(bridged_mock, config)
+
+    ci = first(MOI.get(bridged_mock, MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{Float64}, MOI.GreaterThan{Float64}}()))
+
+    @testset "$attr" for attr in [MOI.ConstraintPrimalStart(), MOI.ConstraintDualStart()]
+        @test MOI.supports(bridged_mock, attr, typeof(ci))
+        MOI.set(bridged_mock, attr, ci, 2.0)
+        @test MOI.get(bridged_mock, attr, ci) ≈ 2.0
+    end
+
+    test_delete_bridge(bridged_mock, ci, 2,
+                       ((MOI.ScalarAffineFunction{Float64},
+                         MOI.Interval{Float64}, 0),
+                       ))
+end
+
+
+@testset "LessToInterval" begin
+    bridged_mock = MOIB.Constraint.LessToInterval{Float64}(mock)
+
+    MOIT.basic_constraint_tests(
+        bridged_mock, config,
+        include = [(F, S)
+                   for F in [MOI.SingleVariable, MOI.ScalarAffineFunction{Float64},
+                             MOI.ScalarQuadraticFunction{Float64}]
+                   for S in [MOI.LessThan{Float64}]])
+
+    MOIU.set_mock_optimize!(mock,
+        (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock,
+            MOI.OPTIMAL, (MOI.FEASIBLE_POINT, [1.0]),
+            MOI.FEASIBLE_POINT,
+                (MOI.ScalarAffineFunction{Float64}, MOI.Interval{Float64}) => [-1.0]
+        ),
+        (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock,
+            MOI.OPTIMAL, (MOI.FEASIBLE_POINT, [2.0]),
+            MOI.FEASIBLE_POINT,
+                (MOI.ScalarAffineFunction{Float64}, MOI.Interval{Float64}) => [-1.0]
+        )
+    )
+    MOIT.solve_set_scalaraffine_lessthan(bridged_mock, config)
+
+    MOIU.set_mock_optimize!(mock,
+        (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock,
+            MOI.OPTIMAL, (MOI.FEASIBLE_POINT, [1.0]),
+            MOI.FEASIBLE_POINT,
+                (MOI.ScalarAffineFunction{Float64}, MOI.Interval{Float64}) => [-1.0]
+        ),
+        (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock,
+            MOI.OPTIMAL, (MOI.FEASIBLE_POINT, [0.5]),
+            MOI.FEASIBLE_POINT,
+                (MOI.ScalarAffineFunction{Float64}, MOI.Interval{Float64}) => [-0.5]
+        )
+    )
+    MOIT.solve_coef_scalaraffine_lessthan(bridged_mock, config)
+
+    ci = first(MOI.get(bridged_mock, MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{Float64}, MOI.LessThan{Float64}}()))
+
+    @testset "$attr" for attr in [MOI.ConstraintPrimalStart(), MOI.ConstraintDualStart()]
+        @test MOI.supports(bridged_mock, attr, typeof(ci))
+        MOI.set(bridged_mock, attr, ci, 2.0)
+        @test MOI.get(bridged_mock, attr, ci) ≈ 2.0
+    end
+
+    test_delete_bridge(bridged_mock, ci, 1,
+                       ((MOI.ScalarAffineFunction{Float64},
+                         MOI.Interval{Float64}, 0),))
+end
+
+
+# FIXME SEGFAULTS
```

Comment:
@blegat I was thinking of adding an unmocked test here, but it's segfaulting. I thought I'd ask if it's ok to add it or not before I try to figure out why it segfaults

Reply:
Yes, the test would be a welcome addition

Reply:
Fixed